persistence/relational-jdbc/src/main/resources/h2/schema-v3.sql

```diff
@@ -129,7 +129,7 @@ CREATE TABLE IF NOT EXISTS events (
     realm_id TEXT NOT NULL,
     catalog_id TEXT NOT NULL,
     event_id TEXT NOT NULL,
```

Comment:
side note: assuming `event_id` values are unique, this table can grow definitely, I guess... Do we have a mechanism to cap its size?

Reply:
I would expect the listener to handle capping the size -- or some other maintenance service responsible for pruning this table.

Reply:
My primary concern is about naive users who deploy pre-built binaries and start accumulating data in this table :thinking: